src/room_member.cpp

```diff
@@ -233,9 +234,13 @@ void RoomMember::ReceiveLoop() {
             peer->DeallocatePacket(packet);
         }
     }
+    peer->Shutdown(300);
```

Comment:
Should only happen if shutdown is `true`.
Currently might also happen as a result of `IsConnected` returning false.
However, that would suggest that the peer is already shut down (otherwise it would be unsafe to signal the user that the connection was closed), so a second shutdown is stupid.